### PR TITLE
redirect ui to ui/

### LINKED ISSUE #5467: UI: {"errors":[]}
I run Vault in dev mode and can not access the UI.

I get {"errors":[]} when I type 127.0.0.1:8200/ui in my browser  

FYI, older .exe of vault works fine

### PR DESCRIPTION
**Pull request by meirish**
fixes #5467

### CODE REVIEW

File: http/handler.go [hunk 1]
```diff
@@ -181,7 +182,7 @@ func wrapGenericHandler(core *vault.Core, h http.Handler, maxRequestSize int64,
 			}
 			r = newR
 
-		case strings.HasPrefix(r.URL.Path, "/ui/"), r.URL.Path == "/":
+		case strings.HasPrefix(r.URL.Path, "/ui"), r.URL.Path == "/":
```
Comment (jefferai) [inline http/handler.go]: Did you actually want this? handlUIRedirect ends up with `/ui/` right?

Comment (meirish) [reply http/handler.go]: Hmm, maybe I should add `r.URL.Path ==  "/ui"` to the case instead? We want to redirect `/ui` as well as `/` here and w/o hitting this case statement the redirect doesn't happen.

Comment (jefferai) [reply http/handler.go]: Ah, this handler is coming before handleUIRedirect. Never mind me, carry on :-)

**Comment by meirish** [comment]
merging as I got an out of band 👍 from @jefferai

**Pull request merged**